packages/fa-icon-chooser/src/components/fa-icon-chooser/fa-icon-chooser.tsx

```diff
@@ -449,7 +455,15 @@ export class FaIconChooser {
   }
 
   isSharpSolidAvailable() {
-    return this.pro() && !!this.resolvedVersion().match('(6.2[0-9]*.)');
+    return this.pro() && !!this.resolvedVersion().match('(6.4[0-9]*.)');
+  }
+
+  isSharpLightAvailable() {
+    return this.pro() && !!this.resolvedVersion().match('(6.4[0-9]*.)');
```

Comment:
Sharp Light was released in 6.4.0. While this regex works for now, it will start failing when we release FA 6.5.0, though it shouldn't.

I suggest instead:

```javascript
semver.satisfies(this.resolvedVersion(), '>=6.4.0')
```